exporter/awsxrayexporter/internal/translator/aws.go

```diff
@@ -155,7 +155,7 @@ func makeAws(attributes map[string]pdata.AttributeValue, resource pdata.Resource
 	}
 
 	// EC2 - hostID/ec2_instanceId should be always returned by ec2 detector
```

Comment:
Should this comment be updated?

Reply:
Hi @codeboten, it is still valid comment. :)

Reply:
The change in the code would indicate that it's ok for the hostID to be `""` in which case the instance ID returned would be `""`, this appears to be conflicting with this comment of always returning a hostID/instanceID, unless `""` is a valid instance ID. This is why I was asking :)

Reply:
Updated the comment to avoid confusion.  Thanks! :)